manifests/bucketeer/charts/batch/values.yaml

```diff
@@ -204,6 +204,49 @@ subscribers:
     maxMps: 1000
     workerNum: 5
 
+onDemandSubscribers:
+  evaluationCountEventDWHPersister:
+    project:
+    topic:
+    subscription:
+    pullerNumGoroutines: 5
+    pullerMaxOutstandingMessages: 1000
+    pullerMaxOutstandingBytes: 1000000000
+    maxMps: 50
+    workerNum: 1
+    checkInterval: 5
+  evaluationCountEventOPSPersister:
+    project:
+    topic:
+    subscription:
+    pullerNumGoroutines: 5
+    pullerMaxOutstandingMessages: 1000
+    pullerMaxOutstandingBytes: 1000000000
+    maxMps: 50
+    workerNum: 1
+    checkInterval: 5
+  goalCountEventDWHPersister:
+    project:
+    topic:
+    subscription:
+    pullerNumGoroutines: 5
+    pullerMaxOutstandingMessages: 1000
+    pullerMaxOutstandingBytes: 1000000000
+    maxMps: 50
+    workerNum: 1
+    checkInterval: 5
+  goalCountEventOPSPersister:
+    project:
+    topic:
+    subscription:
+    pullerNumGoroutines: 5
```

Comment:
Please use the following default settings for all the new persisters.

```yaml
    pullerNumGoroutines: 5
    pullerMaxOutstandingMessages: 1000
    pullerMaxOutstandingBytes: 100000000
    maxMps: 100
    workerNum: 1
    checkInterval: 10
```

Reply:
Updated.